Project.toml

```diff
@@ -5,6 +5,7 @@ version = "0.6.2"
 
 [deps]
 BFloat16s = "ab4f0b2a-ad5b-11e8-123f-65d77653426b"
+DifferentialEquations = "0c46a032-eb83-5123-abaf-570d42b7fbaa"
```

Comment:
Ideally DifferentialEquations.jl shouldn't be added as a dependency as it's a massive package and so would slow down the `using`/`import` time of StochasticRounding.jl drastically. Is it only needed for testing? Then it could be added to `[test]`